mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -539,6 +539,10 @@ class LinearRegressionTrainingSummary private[regression] (
  * Linear regression results evaluated on a dataset.
  *
  * @param predictions predictions outputted by the model's `transform` method.
+ * @param predictionCol Field in "predictions" which gives the predicted value of the label at
+ *                      each instance.
+ * @param labelCol Field in "predictions" which gives the true label of each instance
+ *                 (if available).
```

Comment:
Hmm, good point; "if available" only makes sense for the generic summary classes, not the *TrainingSummary classes.